docs/examples/src/test/java/generic/ContainerNetworkCreationTest.java

```diff
@@ -0,0 +1,43 @@
+package generic;
```

Comment:
I guess this one is not needed anymore :)

Reply:
And nuked! 👍